### PR TITLE
Better logging

### PR DESCRIPTION
**Pull request by mark-i-m**
- Reduce default logging level to INFO, rather than DEBUG
- Send log level ERROR or higher to stderr and everything else to stdout

### CODE REVIEW

**Comment by mark-i-m** [comment]
@PlasmaPower Meritocracy?

File: chaos.py [hunk 1]
```diff
@@ -28,10 +28,36 @@
 import encryption  # noqa: F401
 
 
+class LessThanFilter(logging.Filter):
+    """
+    Source: https://stackoverflow.com/questions/2302315/how-can-info-and-
+    debug-logging-message-be-sent-to-stdout-and-higher-level-messag
```
Comment (PlasmaPower) [inline chaos.py]: Change this to just https://stackoverflow.com/questions/2302315 to avoid wrapping a link

**Review by PlasmaPower — APPROVED** [review]

**Comment by PlasmaPower** [comment]
Sorry I confused this with #475. Reviewed!

File: chaos.py (hunk 1 above)
Comment (mark-i-m) [reply chaos.py]: Oh, I didn't know you could do that. Great!

**Comment by mark-i-m** [comment]
@PlasmaPower I made the fix you proposed...

**Review by PlasmaPower — APPROVED** [review]

**Pull request merged**